lib/app.js

```diff
@@ -1,46 +1,65 @@
-const net = require('net');
```

Comment:
on my end, one of the tests isn't passing giving a net is not defined, but it's probably because these consts were removed!